edbdeploy/terraform.py

```diff
@@ -124,8 +134,10 @@ def apply(self, vars_file):
                     "-auto-approve",
                     "-var-file",
                     vars_file,
-                    "-no-color",
-                    self.dir
+                    "-no-color"
+                    # Before Terraform 0.15.1
+                    #,
+                    #self.dir
```

Comment:
Please remove those commented lines and pass the `TF_DATA_DIR` env. variable through the `environ` argument.